src/common/Footer.js

```diff
@@ -39,14 +50,68 @@ const FooterNavBookmark = styled(NavLink)`
 export default function Footer() {
   return (
     <FooterNav>
-      <FooterNavLink to="/ascending">
-        <FaChevronCircleUp />
-      </FooterNavLink>
-      <FooterNavLink to="/descending">
-        <FaChevronCircleDown />
-      </FooterNavLink>
+      <FooterSortLink
+        to={
+          window.location.pathname === '/'
```

Comment:
Maybe this whole part could be reduced with a function, but that would be something for the last week :)